packages/flutter_tools/gradle/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ gradlePlugin {
             id = "dev.flutter.flutter-gradle-plugin"
             implementationClass = "FlutterPlugin"
         }
+        // The "flutterAppPluginLoaderPlugin" name isn't used anywhere.
```

Comment:
what does this comment mean in practice?
(isn't it used on the very next line, for example?)

Reply:
It means that `flutterAppPluginLoaderPlugin` can be changed to something other (for whatever reason) and it won't affect anything.

Nothing in the codebase references `flutterAppPluginLoaderPlugin`.